app/src/test/java/org/mozilla/tv/firefox/webrender/cursor/CursorViewModelTest.kt

```diff
@@ -163,73 +157,6 @@ class CursorViewModelTest {
         isEnabledTestObs.assertValues(*Array(emittedEventCount) { true })
     }
 
-    @Test
-    fun `GIVEN intended Click event THEN ACTION_UP MotionEvent returns same event_time as ACTION_DOWN`() {
```

Comment:
As we discussed, I would prefer to have a test but I'm currently in the, "why not test everything?" mindset and recognize that it's impractical to test everything. Given how simple it is and how adding a test will only make this code harder to intentionally change, I think it's reasonable to not write a test for this code.